Scrumdinger/Views/DetailView.swift

```diff
@@ -47,10 +50,10 @@ struct DetailView: View {
                     Label("No meetings yet", systemImage: "calendar.badge.exclamationmark")
                 }
                 ForEach(scrum.history) { history in
-                    NavigationLink(destination: HistoryView(history: history                                                                                                                                                                                                                                            )) {
+                    NavigationLink(destination: HistoryView(history: history)) {
                         HStack {
                             Image(systemName: "calendar")
-                            Text(history.date, style: .date)
+                            Text(history.date ?? Date(), style: .date)
```

Comment:
Is it not better to show that a date is not available as opposed to adding a default value?